README.adoc

```diff
@@ -1420,57 +1420,63 @@ def banned?
 end
 ----
 
-=== `and`/`or` [[no-and-or-or]]
+=== `and`/`or` [[no-and-or-or]] [[and-or-flow]]
 
-The `and` and `or` keywords are banned.
-The minimal added readability is just not worth the high probability of introducing subtle bugs.
-For boolean expressions, always use `&&` and `||` instead.
-For flow control, use `if` and `unless`; `&&` and `||` are also acceptable but less clear.
+`and` and `or` are control flow operators. They have very low precedence, and should only be used
+as a final fallback statements.
```

Comment:
Why fallback? I guess you meant final expression in a method in general, right?

Reply:
@bbatsov I tried to adjust the wording, making it more explanatory. I also moved the "good usage" example to be the first one, so it also works as a demo of the concept.